src/sentry/integrations/github/webhook.py

```diff
@@ -1072,7 +1093,8 @@ def handle(self, request: HttpRequest) -> HttpResponse:
             return HttpResponse(status=400)
 
         try:
-            handler = self.get_handler(request.META["HTTP_X_GITHUB_EVENT"])
+            github_event = request.META["HTTP_X_GITHUB_EVENT"]
```

Comment:
This is the real way we can distinguish events.

Yesterday I regressed `check_run` when I started looking at an event_type of `ci_run`.